SPECIFICATION.md

```diff
@@ -1,57 +1,117 @@
 # Chia Pool Protocol 1.0
 
 This is the initial version of the Chia Pool Protocol. It has been designed to be simple, and to be extended later.
-It relies on farmers having smart coins (referred to as Pool NFT in GUI + CLI) on the blockchain which allow them to dynamically set their pool to a different
-pool by making a blockchain transaction. Furthermore, it decreases the reliance on pools for block production, since
+It relies on farmers having smart coins (referred to as Pool NFT in GUI + CLI) which allow them to switch between pools
+by making transactions on the blockchain. Furthermore, it decreases the reliance on pools for block production, since
 the protocol only handles distribution of rewards, and it protects against pools or farmers acting maliciously.
 
 ## Parties
 
-The parties involved in the pool protocol involve the pool operator and farmers. Each farmer is running
+The parties involved in the pool protocol are the pool operator and farmers. Each farmer is running
 a farmer process, and any number of harvester processes connected to that farmer process. The full node can either be
 run by the farmer (the default in the Chia GUI application), or run by the pool operator. If the farmer does not want
 to run a full node, they can configure their node to connect to a remote full node.
 
-A pool operator can support any number of farmers. 
+A pool operator can support any number of farmers.
+
+## Farmer identification
```

Comment:
We should add a few sentences to make explicit the difference between this part of the pool protocol, and the optional `/login` below.

I think `/login` is described well enough once they get down that far in the document.